plugins/chat/spec/system/single_thread_spec.rb

```diff
@@ -20,7 +20,7 @@
 
     before { channel.update!(threading_enabled: false) }
 
-    it "does not open the side panel for a single thread" do
+    it "does not open the side panel for a single thread", capture_log: true do
```

Comment:
This test had a DB transaction problem in 

https://github.com/discourse/discourse/actions/runs/7259074759/job/19775524941